app/src/main/java/cm/aptoide/pt/analytics/Analytics.java

```diff
@@ -832,4 +832,32 @@ public static void installationType(boolean isRootAllowed, boolean isRoot) {
       logFabricEvent(IS_INSTALLATION_TYPE_EVENT_NAME, map, FABRIC);
     }
   }
+
```

Comment:
Extract this inner class to your module/flavour following the structure used for AppViewAnalytics.